### PR TITLE
thuang-126-CZIF-policy-update

### PR DESCRIPTION
**Pull request by tihuan**
https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/chanzuckerberg/single-cell/126

NOTE: Explorer cookie banner will be updated in a separate PR. Thank you!

#### Reviewers
**Functional:** 
@brianraymor 

**Readability:** 
@MDunitz 

Live site: https://thuang-frontend.rdev.single-cell.czi.technology/previewpolicies/
---

## Changes
- add
- Add PreviewPolicies page `/previewpolicies/`

Screen Shot 2021-03-15 at 12 14 14 PM

- modify
- Update ToS page

Screen Shot 2021-03-15 at 12 16 51 PM

- Update Privacy page

  **UPDATE (3/15: 1:16pm)**
Screen Shot 2021-03-15 at 1 17 04 PM

- Update cookie banner to add `(updates coming)`

Screen Shot 2021-03-15 at 12 03 50 PM

- Update footer to add `(updates coming)` to ToS and Privacy links

Screen Shot 2021-03-15 at 12 04 06 PM

## QA steps (optional)

PTAL thanks all!

### CODE REVIEW

File: frontend/src/pages/privacy.tsx [hunk 1]
```diff
@@ -31,12 +31,18 @@ const ToS: FC = () => {
...
+
             <p>
               April 1, 2020. There is no prior version of this privacy policy.
             </p>
 
             <h2>Introduction</h2>
 
+            <p>
+              We invite you to <a href="/previewpolicies">preview</a> our
+              updated Terms of Use, which will go into effect on April 1, 2021.
```
Comment (brianraymor) [inline frontend/src/pages/privacy.tsx:43]: Looks like a cut-n-paste error - this is the privacy policy, not terms of use:

> We invite you to preview [link to cellxgene.cziscience.com/previewpolicies] our updated **Privacy Policy,** which will go into effect on April 1, 2021.

Comment (tihuan) [reply frontend/src/pages/privacy.tsx:43]: Oh!! Yes yes 🙏 Thank you for the eagle eye, @brianraymor ! Will update now! 🦅

Comment (tihuan) [reply frontend/src/pages/privacy.tsx]: Updated! Thanks again!

Screen Shot 2021-03-15 at 1 15 14 PM

**Review by MDunitz — APPROVED** [review]
lgtm

**Comment by tihuan** [comment]
Thanks Madison 🎉 !! We actually got the rest of the content from Shawn today, so I'm adding the content to this PR before merging now instead of doing two PRs 😄 !

**Comment by tihuan** [comment]
@brianraymor @MDunitz just added a live site for this change 🤩 PTAL!

https://thuang-frontend.rdev.single-cell.czi.technology/previewpolicies/

Thank you!

**Pull request merged**